src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculationInterface, FigureDataInterface {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The 'draw()' and 'getArea()' methods are not conceptually close to each other, so they should not be in the same interface.